src/migrations/1584619633448_txs.ts

```diff
@@ -157,26 +157,21 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
 
   });
 
-  pgm.createIndex('txs', 'tx_id');
```

Comment:
This one is surprising. There's several queries against the `txs` table using the `tx_id` column as a primary search condition. I can definitely see how removing the index would speed up writes, but seems like this one would dramatically slow slow down a bunch of reads?

Reply:
I actually validated this using `EXPLAIN ANALYZE`, and that index is just substituted by the `unique_tx_id_index_block_hash_microblock_hash` index when querying by `tx_id`, example:
```
Index Scan using unique_tx_id_index_block_hash_microblock_hash on txs  (cost=0.55..8.57 rows=1 width=1585) (actual time=2.181..2.193 rows=1 loops=1)
"  Index Cond: (tx_id = '\x4510e6cff6414747f402126b92e35d90165f9f5e1f2750f50ed2b8e052b25108'::bytea)"
Planning Time: 0.159 ms
Execution Time: 2.239 ms
```

Reply:
That's cool! I wasn't aware constraints could be used as indexes like that, makes sense though!